sdk/servicebus/azure-servicebus/azure/servicebus/aio/_transport/_pyamqp_transport_async.py

```diff
@@ -169,6 +170,13 @@ def create_receive_client_async(
         config = receiver._config   # pylint: disable=protected-access
         source = kwargs.pop("source")
         receive_mode = kwargs.pop("receive_mode")
+        link_properties = kwargs.pop("link_properties")
+
```

Comment:
since this seems like just a pyamqp change, I believe we can just create/add/encode the operation timeout as a uint here, rather than in the receiver.

You can get the session_id and max_wait_time attributes by calling them on the `receiver` here.